iml-gui/crate/src/components/tree.rs

```diff
@@ -752,6 +756,60 @@ mod tests {
         )
     }
 
+    #[test]
+    fn test_cache_cloning() {
+        let c1 = fixtures::get_cache();
+        let mut c2: Cache = c1.clone();
+        let mut c3: Cache = c2.clone();
+
+        // The hashmap ost_pool_osts is the same for all cache clones c1, c2, c3
+        assert_eq!(
+            &*c1.ost_pool_osts as *const HashMap<_, _>,
+            &*c2.ost_pool_osts as *const HashMap<_, _>
+        );
+        assert_eq!(
+            &*c1.ost_pool_osts as *const HashMap<_, _>,
+            &*c3.ost_pool_osts as *const HashMap<_, _>
+        );
+
+        let rec1 = OstPoolOstsRecord {
+            id: 1,
+            ostpool_id: 1,
+            managedost_id: 1,
+        };
+        let rec2 = OstPoolOstsRecord {
+            id: 2,
+            ostpool_id: 2,
+            managedost_id: 2,
+        };
+        let rec18 = OstPoolOstsRecord {
+            id: 18,
+            ostpool_id: 18,
+            managedost_id: 13,
+        };
+        let rec19 = OstPoolOstsRecord {
```

Comment:
This and rec18 look like a copy of the fixture. I think it is better to have only one source of truth.

Reply:
Yep, agree.